src/NzbDrone.Core.Test/Profiles/Release/ReleaseProfileServiceFixture.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections.Generic;
+using System.Linq;
+using FizzWare.NBuilder;
+using FluentAssertions;
+using NUnit.Framework;
+using NzbDrone.Core.Profiles.Releases;
+using NzbDrone.Core.Test.Framework;
+
+namespace NzbDrone.Core.Test.Profiles
+{
+    [TestFixture]
+    public class ReleaseProfileServiceFixture : CoreTest<ReleaseProfileService>
+    {
+        private List<ReleaseProfile> _releaseProfiles;
+        private ReleaseProfile _defaultReleaseProfile;
+        private ReleaseProfile _includedReleaseProfile;
+        private ReleaseProfile _excludedReleaseProfile;
+        [SetUp]
+        public void Setup()
+        {
+            _releaseProfiles = Builder<ReleaseProfile>.CreateListOfSize(3).TheFirst(1)
+                .With(r => r.Required = ["required_one"])
+                .TheNext(1)
+                .With(r => r.Required = ["required_two"])
+                .TheNext(1)
+                .With(r => r.Required = ["required_three"])
+                .Build()
+                .ToList();
+
+            _defaultReleaseProfile = _releaseProfiles[0];
+            _includedReleaseProfile = _releaseProfiles[1];
+            _includedReleaseProfile.Tags.Add(1);
+            _excludedReleaseProfile = _releaseProfiles[2];
+            _excludedReleaseProfile.ExcludedTags.Add(2);
+
+            Mocker.GetMock<IRestrictionRepository>()
+                  .Setup(s => s.All())
+                  .Returns(_releaseProfiles);
+        }
+
+        [Test]
+        public void untagged_artist_should_match_default_and_excluded_release_profiles()
```

Comment:
```suggestion
        public void untagged_series_should_match_default_and_excluded_release_profiles()
```

Reply:
Thanks, I have changed it to `all_for_tags_should_match_default_and_excluded_release_profiles_for_untagged_series` to better match the other test names.